src/CloudTasksServiceProvider.php

```diff
@@ -29,8 +29,13 @@ public function boot(): void
 
     private function registerClient(): void
     {
-        $this->app->singleton(CloudTasksClient::class, function () {
-            return new CloudTasksClient();
+        $this->app->singleton(CloudTasksClient::class, function ($app) {
+            $config = config('queue.connections.cloudtasks');
+            $options = [];
+            if (!empty($config['credential_file'])) {
```

Comment:
@miccehedin This can already be done by setting the `GOOGLE_APPLICATION_CREDENTIALS` environment. Can I therefore remove this bit?